src/Decompiler/Decompiler.cs

```diff
@@ -414,6 +414,37 @@ public void WriteDecompiledTypes(Program program, string headerFilename, TextWri
                     w.WriteLine();
                 }
             }
+            WriteExternalProcedures(program, w);
+        }
+
+        public static void WriteExternalProcedures(
+            Program program, TextWriter w)
+        {
+            var externalProcedures = program.ExternalProcedures.OrderBy(
+                p => (
+                    p.Value.Item1,
+                    p.Value.Item3.EnclosingType?.ToString(),
+                    p.Value.Item3.Name)
+            ).ToList();
+            if (externalProcedures.Count == 0)
+                return;
+            w.WriteLine("// Declarations for external procedures");
+            w.WriteLine();
+            foreach (var epKv in externalProcedures)
```

Comment:
Suggestion: you can write the header of the `foreach` loop like this:
```C#
foreach (var (importName, (dllName, convention, ep)) in externalProcedures)
...
```

Reply:
Done in 636607b